hooks/command

```diff
@@ -13,60 +13,121 @@ fi
 ((timeout=BUILDKITE_TIMEOUT*60))
 export BUILDKITE_TIMEOUT
 
+# Default values can be overridden by setting "K8S_PLUGIN_*" env vars as used below.
+readonly job_apply_loop_interval_seconds="${K8S_PLUGIN_JOB_APPLY_LOOP_INTERVAL:-5}"
+readonly job_apply_loop_timeout_seconds="${K8S_PLUGIN_JOB_APPLY_LOOP_TIMEOUT:-120}"
```

Comment:
For the added configs, could we use the format `*RETRY_INTERVAL_MS` and `*TIMEOUT_MS` in place of `*LOOP_INTERVAL` and `LOOP_TIMEOUT` respectively? dropping `loop` from their names and adding time resolution since that could be more descriptive of what they're for.